packages/core/src/scuttle.js

```diff
@@ -22,6 +22,14 @@ const {
   defineProperty,
 } = Object
 
+const preserveLanguageIntrinsics = (function(cg, g) {
```

Comment:
if `function` is used here intentionally and cannot be substituted by an arrow expression, please explain why in a comment

Reply:
is there a good argument for using arrow exp over function that I'm missing?

Reply:
The only real argument for this is that if the function won't be called with a context and/or it does not reference a context, then you don't need `function` (similar argument for when to use [method definitions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions)).  

Otherwise it's just a matter of consistent style and ease of future refactors. For example, Mocha always executes callbacks with a context, so it's recommended to use `function` when defining a suite/test/hook callback even if you're not currently using `this` (though plenty disregard this advice, as it's plainly more characters to type 🙂).

Determining whether that's a _good_ argument or not is an exercise left to the reader.